packages/kit/src/runtime/server/page/fetch.js

```diff
@@ -68,6 +68,13 @@ export function create_fetch({ event, options, state, route, prerender_default,
 
 				const url = new URL(request.url);
 
+				if (
+					!request.headers.get('origin') &&
```

Comment:
In https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin#description it says "if a cross-origin GET or HEAD request is made in no-cors mode, the Origin header will not be added" - should we add that check here, or is this irrelevant here?